src/app/inventory/store/d2-item-factory.service.js

```diff
@@ -242,8 +242,9 @@ export function D2ItemFactory(
       dmg: dmgName,
       visible: true,
       lockable: item.lockable,
-      tracked: item.state === 2,
-      locked: item.state === 1,
+      tracked: item.state & 2,
+      locked: item.state & 1,
+      masterwork: item.state & 4,
```

Comment:
Whoa, didn't realize this was a bitfield!

Reply:
Yeah, vthorn mentioned it in the dev channel so I tidied that up.

I didn't see it being used anywhere in my items, so it might be purely theoretical.